lib/modulesync.rb

```diff
@@ -183,9 +184,14 @@ def self.update(options)
     exit 1 if errors && options[:fail_on_warnings]
   end
 
-  def self.pr(module_options = {})
-    github_conf = module_options[:github]
-    gitlab_conf = module_options[:gitlab]
+  def self.pr(module_options)
```

Comment:
Why is it needed to change the signature?

Reply:
The changes to the code makes the default value unnecessary (and potentially misleading). We _could_ leave it in, but it's technically unneeded, I believe.